charts/refinery/templates/ingress-grpc.yaml

```
@@ -24,8 +24,8 @@ spec:
       secretName: {{ .secretName }}
     {{- end }}
   {{- end }}
-  {{- if .Values.ingress.className }}
-  ingressClassName: {{ .Values.ingress.className }}
+  {{- if .Values.grpcIngress.ingressClassName }}
```

Overall I like the API change! We do have to make sure it is backwards compatible here in order for this not to be a breaking change so we should still check for the old value

```suggestion
  {{- if or .Values.grpcIngress.ingressClassName .Values.ingress.className }}
```